harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -649,6 +655,40 @@ def _load(self) -> None:
             for idx, lr_scheduler in enumerate(self.context.lr_schedulers):
                 lr_scheduler.load_state_dict(checkpoint["lr_schedulers_state_dict"][idx])
 
+        if "amp_state" in checkpoint:
+            if self.context._use_amp:
+                apex.amp.load_state_dict(checkpoint["amp_state"])
+            else:
+                logging.warn(
```

Comment:
Change all of these to `logging.warning()`, apparently `logging.warn()` is not the preferred alias.